src/components/Menu/MenuItem.scss

```
@@ -45,4 +50,26 @@
 
         background: none;
     }
+
+    &.moonstone-menuItem-withImage_small {
+        height: $menuItem-withImage-height_default;
+        padding-left: 16px;
+    }
+
+    &.moonstone-menuItem-withImage_big {
+        height: $menuItem-withImage-height_big;
+        padding-left: 16px;
+    }
```

You can use spacing tokens for padding
```suggestion
    &.moonstone-menuItem-withImage_small {
        height: $menuItem-withImage-height_default;
        padding-left: var(--spacing-medium);
    }

    &.moonstone-menuItem-withImage_big {
        height: $menuItem-withImage-height_big;
        padding-left: var(--spacing-medium);
    }
```